### ISSUE TITLE
Lobby: notifications optimization

### ISSUE THREAD

**Issue by leafcutterant** [darkwallet/darkwallet#54]
- [ ] Both popups (white and blue) show up when the tab is active, while the white one is only needed when the tab is not active.
- [ ] The white pop-up tells too little. Besides the lobby group, it could include the "funny pixel" avatar and the cloak name of the sender.

From other issues:
- [ ] Lobby notification pops up when one posts their own comments
- [x] Notifications service only receives notifications from a single channel

**Comment by sembrestels** [comment]
Yes, we have to connect frontend and backend notifications, and do not show the desktop (backend) one when the frontend one is shown.

Good idea add the cloack image in the backend notification!

**Comment by caedesvvv** [comment]
Also shouldn't show a notification when it's your own comment.
